### PR TITLE
add video stickers

### PR DESCRIPTION
**Pull request by akhrail1996**

### CODE REVIEW

File: lib/t_web/views/view_helpers.ex [hunk 1]
```diff
@@ -69,7 +69,19 @@ defmodule TWeb.ViewHelpers do
           |> Enum.reduce([], fn label, acc ->
             case label do
               %{"s3_key" => key, "question" => "audio"} ->
-                label = Map.put(label, "url", Accounts.media_url(key))
+                label = Map.put(label, "url", media_cdn_url(key))
+                [label | acc]
+
+              %{
+                "s3_key" => key,
+                "placeholder_s3_key" => placeholder_s3_key,
```
Comment (ruslandoga) [inline lib/t_web/views/view_helpers.ex:77]: `placeholder_s3_key` -> `s3_key`, `s3_key` -> `video_s3_key` to be consistent with video backgrounds?

File: lib/t_web/views/view_helpers.ex [hunk 2]
```diff
@@ -51,7 +50,7 @@ defmodule TWeb.ViewHelpers do
     end
   end
 
-  defp process_labels(%{"labels" => labels} = page, version) do
+  defp process_labels(%{"labels" => labels} = page, version, screen_width) do
     # we do not add url for versions prior to 6.2.0 because they will be rendered incorrectly
     labels =
       case Version.compare(version, "6.2.0") do
```
Comment (ruslandoga) [inline lib/t_web/views/view_helpers.ex:56]: needs another version check, maybe against v6.3.0?

**Review by ruslandoga — APPROVED** [review]

File: lib/t_web/views/view_helpers.ex (hunk 1 above)
Comment (akhrail1996) [reply lib/t_web/views/view_helpers.ex:77]: right

File: lib/t_web/views/view_helpers.ex (hunk 2 above)
Comment (akhrail1996) [reply lib/t_web/views/view_helpers.ex:56]: may do it, but not sure if I should since the "video" (unknown question) is "rendered" to nil in 6.2.0

**Pull request merged**